OWNERS

```diff
@@ -1,15 +1,13 @@
 reviewers:
+  - alaypatel07
   - deads2k
```

Comment:
pull me from the regular reviewers. I'll come in for approval or if you need me.